packages/x-components/src/x-modules/facets/service/__tests__/facets-service.spec.ts

```diff
@@ -575,25 +582,57 @@ describe('testing facets service', () => {
         id: 'backend',
         facets: [newSizeFacet]
       });
+
+      expect(getSelectedFilters()).toHaveLength(1);
```

Comment:
Should we check if the selected is the correct one (in our case the last one)?

`expect(getSelectedFilters()).toStrictEqual([newSizeFacet.filters[2]]);`

Reply:
This test is just to ensure that only one filter is selected. I don't think the "which one" matters, as the outcome is that the app keeps working properly.